llama_hub/jira/base.py

```diff
@@ -60,7 +60,17 @@ def __init__(
             )
 
     def load_data(self, query: str) -> List[Document]:
-        relevant_issues = self.jira.search_issues(query)
+        relevant_issues = []
+        start_at = 0
+        max_results = 50
```

Comment:
could this be a configurable parameter in the `__init__` instead of hardcoded? can default to 50

Reply:
done!